README.md

```diff
@@ -314,6 +314,49 @@ overrides the value specified in **Workspace**.
 | `independent` | `bool` | No | Yes | `version`, `publish` |
 | `no_individual_tags` | `bool` | Yes | No | `version`, `publish` |
 
+## Running tests
+
+The recommended way to run tests for the crate is as follows:
+
+```sh
+cargo test --manifest-path cargo-workspaces/Cargo.toml -j1
```

Comment:
I'm not fully sure if that's an appropriate fix, but it seems to work consistently.
Without it, tests were always failing. Probably [this issue](https://github.com/palfrey/serial_test/issues/100) may be related.